deploy/roles/batyam/tasks/main.yml

```diff
@@ -1,2 +1,58 @@
 ---
 # tasks file for batyam
+
+- name: install openshift package for the 'k8s' module
+  pip:
+    name: openshift
+
+- name: Create configmap batyam-gerrit-identity-path-envvar
+  k8s:
+    api_key: "{{ batyam_ocp_token }}"
+    host: "{{ batyam_ocp_host }}"
+    state: "{{ batyam_deploy_state }}"
+    #this is currently commented out as I didn't test with a CACERT in place
+    #ca_cert: "{{ batyam_ca_cert }}"
+    name: batyam-gerrit-identity-path-envvar
+    namespace: "{{ batyam_ocp_resources_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        # maybe add prefix of 'batyam-' to make resources use better
+        #
+        # a good convention would be adding envvar to anything we import
+        # as an envvar to have the '-envvar' suffix
+        name: batyam-gerrit-identity-path-envvar
+        namespace: "{{ batyam_ocp_resources_namespace }}"
+      data:
+        # this can also split up into multiple defaults
+        #
+        # # default_path
+        # folder name we decided on that can be hardcoded
+        # # gerrit_token_key_name
+        GERRIT_IDENTITY_FILE: /tmp/gerrit-key
+
+- name: clone a private repository
+  git:
+    repo: "{{ batyam_configs_repo }}"
+    key_file: "{{ batyam_configs_repo_key }}"
+    dest: "{{ batyam_configs_repo_dir }}"
+    version: "{{ batyam_configs_repo_version }}"
+
+- name: create batyam-repos-volume configmap
+  k8s:
+    api_key: "{{ batyam_ocp_token }}"
+    host: "{{ batyam_ocp_host }}"
+    state: "{{ batyam_deploy_state }}"
+    # a good convention would be adding volume to anything we import
+    # as an volume to have the '-volume' suffix
+    name: batyam-repos-volume
+    namespace: "{{ batyam_ocp_resources_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: batyam-repos-volume
+        namespace: "{{ batyam_ocp_resources_namespace }}"
+      data:
+        config.yaml: "{{ lookup('file', batyam_configs_repo_dir + '/config.yaml') }}"
```

Comment:
found this
```
{{ ('/etc', path, 'subdir', file) | path_join }}
```
in https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#id40
I will test and see if it's good

Reply:
I am not following what this can help.

Reply:
This can help in case of win/linux filepaths I think

Reply:
Is there a request to handle that now? If not, do not over engineer this. We can consider that in the future if such a demand comes along.